src/dune_rules/melange.mli

```diff
@@ -22,3 +45,16 @@ module Cm_kind : sig
     val make_all : 'a -> 'a t
   end
 end
+
+val lib_output_dir :
+     melange_stanza_dir:Path.Build.t
+  -> lib_dir:Path.Build.t
```

Comment:
What's the `lib_dir`? If it's the directory where the library is defined, then it should be `lib_src_dir` I suppose.